### ISSUE TITLE
Tracking - looking at cards in the deck

### ISSUE THREAD

**Issue by jay1148** [Cardshifter/Cardshifter#268]
Can we implement "tracking" where we can rearrange the order of the top 3 cards in our deck, or an opponents deck; as well as discard a card in an opponents deck?

**Comment by jay1148** [comment]
Also, choosing or having your opponent choose a card in the hand to discard.

**Comment by Zomis** [comment]
Yes, this is planned. This is a bit tricky as it involves some GUI. The way I will accomplish this I think is by temporarily creating a card zone that those cards will be put in, and then show that zone to the player and have him choose what to do on those cards.
